src/db.rs

```diff
@@ -519,6 +549,10 @@ impl DbInner {
 		if let Some((record_id, cleared, bytes)) = cleared {
 			self.log.end_read(cleared, record_id);
 			{
+				// [Review] when validation_mode is true we still process
+				// the queue. Logic here is that validation mode is not use
+				// from queue but on restart, maybe consider renaming 'validation_mode'
+				// to 'replay_mode'.
```

Comment:
> when validation_mode is true we still process the queue

Not sure what you mean here. `validation_mode` The commits that are processed with `validation_mode` are not in the commit queue.

Reply:
I was mentioning the items from the log (referring to the log as queue was incorrect). Just the fact that the expresion folowing 'validation' if branch are processed (no 'else' statement).

Reply:
OK, I guess it can be renamed